src/tabs/gps.html

```diff
@@ -135,7 +142,13 @@
                     </div>
                     <div id="loadmap">
                         <div id="map" class="map"> </div>
+
                         <div class="controls">
+                            <div id="mapview" class="mapview">
+                                <a href="#" id="Hybrid">H</a>
+                                <a href="#" id="Satellite">S</a>
+                                <a href="#" id="OSM">R</a>
```

Comment:
This kind of drives me nuts. Why are we using links for what's clearly a button?

Reply:
Not sure - just added to the zooming `buttons`.

Reply:
Happy to learn a better way - but our code is full of this pattern:
```
<div class="buttons">
    <a href="#" class="someClass [optionalClass]" i18n="somethingToTranslate">
</div>
```

Reply:
Added a value to see if I could read it using `e.target.value` but it was either null or undefined?

Reply:
Fixed

Reply:
https://stackoverflow.com/questions/18314019/anchor-or-button